torch/package/package_exporter.py

```diff
@@ -550,7 +549,10 @@ def _intern_module(
                 self.add_dependency(dep)
 
     def save_pickle(
-        self, package: str, resource: str, obj: Any, dependencies: bool = True
+        self, package: str,
+        resource: str, obj: Any,
+        dependencies: bool = True,
+        save_mocked_and_used: bool = False
```

Comment:
I think we don't need to make this an option for users. While this change is technically BC-incompatible, any package that trips this behavior would not have been importable anyway, so it's probably fine.

Reply:
Yeah it was primarily for the unit test, I couldn't think of a better way to do the bc test. I'll just delete it then.